docs/suspensive.org/src/pages/docs/jotai/installation.ko.mdx

```diff
@@ -0,0 +1,9 @@
+# 설치하기
+
+@suspensive/jotai는 npm에 있습니다. 최신 안정버전을 설치하기 위해 아래 커맨드를 실행하세요
+
+[![npm version](https://img.shields.io/npm/v/@suspensive/jotai?color=000&labelColor=000&logo=npm)](https://www.npmjs.com/package/@suspensive/jotai)
+
+```shell npm2yarn
+npm install @suspensive/jotai
+```
```

Comment:
<img width="578" alt="스크린샷 2024-07-17 오후 3 52 45" src="https://github.com/user-attachments/assets/caf672ce-bd4c-47e9-b3e5-4c982727053d">

It looks like maintaining consistency within the document, as shown in the following picture, would be beneficial.